spec/karma.conf.js

```diff
@@ -90,7 +90,7 @@ module.exports = function (config) {
 		// - Opera
 		// - Safari (only Mac)
 		// - IE (only Windows)
```

Comment:
Let's remove this line while we are at it, we no longer support testing in Internet Explorer.
```suggestion
```